### PR TITLE
Updating documentation for Redis Insight

### PR DESCRIPTION
**Pull request by ViktarStarastsenka**

### CODE REVIEW

File: content/develop/connect/insight/install-and-configure/_index.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+---
+categories:
+- docs
+- operate
+- redisinsight
+description: 'Install and configure desktop Redis Insight or Redis Insight on Docker.
```
Comment (mich-elle-luna) [inline content/develop/connect/insight/install-and-configure/_index.md:6]:
```suggestion
description: Install and configure desktop Redis Insight or Redis Insight on Docker.
```

**Review by mich-elle-luna — CHANGES_REQUESTED** [review]
thank you! Let's discuss this a bit further when David gets back next week

File: content/develop/connect/insight/install-and-configure/_index.md (hunk 1 above)
Comment (dwdougherty) [reply content/develop/connect/insight/install-and-configure/_index.md:6]: I've removed @mich-elle-luna's comment. The leading single quote is required here.

**Comment by dwdougherty** [comment]
We're not going to consolidate all the RI content as was done here. There will be a new pull request that incorporates the desktop changes and re-ordering of topics.

**Pull request closed without merging**